apps/mull-ui/src/utilities.ts

```diff
@@ -5,14 +5,22 @@
  */
 export const formatDate = (
   date: Date
-): { year: number; month: string; day: number; time: string } => {
+): { year: number; month: string; day: number; time: string; merīdiem: string } => {
   const month = Intl.DateTimeFormat('en-us', {
     month: 'short',
   }).format(date);
   const timeString = Intl.DateTimeFormat('en-us', {
     hour: '2-digit',
     minute: '2-digit',
-  }).format(date);
-
-  return { year: date.getFullYear(), month, day: date.getDate(), time: timeString };
+  })
+    .format(date)
+    .split(' ');
+  timeString.push(''); //to take the 24h time into account
```

Comment:
One thing that is kinda dangerous with this is that we can't know what `Intl.DateTimeFormat`, it depends on the format accepted in the region the user lives in. This might have edge cases we aren't aware of

Reply:
This has been addressed